components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -43,9 +43,10 @@ export class CreateObjectModalComponent implements OnInit, OnChanges {
 
   ngOnChanges(changes: SimpleChanges) {
     // if a new list of projects to populate dropdown with is passed in we update the dropdown
+    const checked = false;
     if (hasIn('assignableProjects.currentValue', changes)) {
       changes.assignableProjects.currentValue.forEach(proj =>
-        this.projects[proj.id] = projectCheckedFromProject(proj, false));
+        this.projects[proj.id] = { ...proj, checked });
```

Comment:
[nit] you could also do `{ ...proj, checked: false }`, you don't _need_ to introduce a constant for this.